externs/ngeox.js

```diff
@@ -125,6 +126,14 @@ ngeox.profile.ProfileOptions.prototype.lightXAxis;
 
 
 /**
+ * Return a ratio value for constraining the y domain based on the maximum
+ * value from the x domain.
+ * @type {function(number):(number|undefined)|undefined}
```

Comment:
What does `undefined` mean as a return value? Could we use a negative number instead?